scripts/V1/deployContractsProxy.ts

```diff
@@ -1,10 +1,9 @@
 import { ethers } from "hardhat";
 import { Wallet, NonceManager, Signer } from "ethers";
 import { DeployUtils } from "@/utils";
-import dotenv from "dotenv";
+import peer3Config from "../../peer3config.json";
```

Comment:
Here it's fine to use it like this since this is the deployment script of the SDK (not an external project/example)